src/Controller/ModulesController.php

```diff
@@ -205,11 +189,10 @@ public function uname($id): ?Response
         try {
             $response = $this->apiClient->get(sprintf('/objects/%s', $id));
         } catch (BEditaClientException $e) {
-            if ($e->getCode() === 404) {
-                $error = sprintf(__('Resource "%s" not found', true), $id);
-            } else {
-                $error = sprintf(__('Resource "%s" not available. Error: %s', true), $id, $e->getMessage());
-            }
+            $msg = $e->getMessage();
```

Comment:
[super-minor] we usually prefer `if/else` over ternary operator because it's easier to debug ... not a big deal here...

Reply:
if/else create 2 paths, bad for coverage, good for readability. I choosed better coverage over readability